tests/sklearn/test_dump_onnx.py

```diff
@@ -222,8 +221,15 @@ def test_dump(
   %/_operators.0/Reshape_2_output_0 = Reshape[allowzero = 0](%/_operators.0/Equal_output_0, %/_operators.0/Constant_2_output_0)
   %/_operators.0/MatMul_1_output_0 = MatMul(%_operators.0.weight_3, %/_operators.0/Reshape_2_output_0)
   %/_operators.0/Reshape_3_output_0 = Reshape[allowzero = 0](%/_operators.0/MatMul_1_output_0, %/_operators.0/Constant_3_output_0)
-  %transposed_output = Transpose[perm = [2, 1, 0]](%/_operators.0/Reshape_3_output_0)
-  return %transposed_output
+  """
+        + (
+            """%/_operators.0/ReduceSum_output_0 = ReduceSum[keepdims = 0](%/_operators.0/Reshape_3_output_0, %onnx::ReduceSum_22)
+  %transposed_output = Transpose[perm = [1, 0]](%/_operators.0/ReduceSum_output_0)
+  """
+            if os.getenv("TREES_USE_FHE_SUM") == "1"
```

Comment:
the problem is that the test will only run for one value, not both right ?

Reply:
yes only one

Reply:
well it's not great for coverage no ?

Reply:
It"s not a problem for the coverage, because it's the src that needs to be covered.

on the other hand, it would be really good to relaunch the weekly ci with TREES_USE_FHE_SUM enabled.

Reply:
yeah I meant coverage in a general sense indeed, in this current state we are not testing with `TREES_USE_FHE_SUM` activated, which is a problem I think (and since we're not executing anything , only compiling, it's fine if we do that in regular CIs I think)